schema_source/aws_serverless_api.py

```diff
@@ -131,6 +131,7 @@ class Route53(BaseModel):
     HostedZoneId: Optional[PassThroughProp] = route53("HostedZoneId")
     HostedZoneName: Optional[PassThroughProp] = route53("HostedZoneName")
     IpV6: Optional[bool] = route53("IpV6")
+    RoutingPolicy: Optional[str] = route53("RoutingPolicy")
```

Comment:
You can remove the assignment, we'll add it once https://github.com/awsdocs/aws-sam-developer-guide is updated (otherwise it'll show an empty description in the schema):

```suggestion
    RoutingPolicy: Optional[str]  # TODO: add docs
```